src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -312,5 +312,32 @@ public void TestMailDeliveryAsync()
 
             Assert.True(task.Wait(1000));
```

Comment:
> Sounds like a good suggestion (not sure about the common timeout though). Can you please file a 'test enhancement' issue?

It's common practice and a good pattern to use a named const instead of adding a magic value such as `1000` in several places. I don't think it's worth opening a separate issue.